### PR TITLE
Doc: updates following Rust compilation

### PR DESCRIPTION
**Pull request by WardBrian**
This is a follow on to #212 with a few extra tweaks to the documentation.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-======================================
+BridgeStan -- efficient, in-memory access to the methods of a Stan model
+========================================================================
 
-BridgeStan provides efficient in-memory access through Python, Julia,
-R, and Rust to the methods of a `Stan <https://mc-stan.org>`__ model, including
-log densities, gradients, Hessians, and constraining and unconstraining
-transforms.
+BridgeStan is a library (with :doc:`bindings <languages>` available in Python,
+Julia, R, and Rust) that grants to the methods of a `Stan <https://mc-stan.org>`__
```
Comment (roualdes) [inline docs/index.rst:6]: "grants to" -> "grants access to"

File: docs/languages.rst [hunk 2]
```diff
@@ -8,10 +8,10 @@ written in C.
...
 All language interfaces expose the same core functionality from the
 :doc:`C API <./languages/c-api>`.
-Additional "quality of life" features such as the ability to download
-BridgeStan's source code automatically or compile models from inside the language
-(rather than manually calling :command:`make`) are available on a best-effort basis.
-If you are missing these features in your favorite language, we would welcome a
+
+
+Additional "quality of life" features are available on a best-effort basis.
+If you are missing a feature from one interface in your favorite language, we would welcome a
```
Comment (roualdes) [inline docs/languages.rst:14]: > If you are missing a feature from one interface in your favorite language, we would welcome a contribution 

Can we drop the word would?

I'm also hesitant about the word one.  Maybe the?  Other thoughts?

File: docs/languages/rust.rst [hunk 3]
```diff
@@ -14,9 +14,12 @@ The BridgeStan Rust client is available on `crates.io <https://crates.io/crates/
 
     cargo add bridgestan
 
-The first time you compile a model, the BridgeStan source code will be downloaded to `~/.bridgestan`. If you prefer to use a source distribution of BridgeStan, you can pass its path as the `bs_path` argument to `compile_model`.
+A copy of the BridgeStan C++ sources is needed to compile models. This can be downloaded to
```
Comment (roualdes) [inline docs/languages/rust.rst:17]: "can be" -> "is"

**Review by roualdes — APPROVED** [review]
Definitely throwing some pedantry at you, here.  I'd also be content if you ignore all my grammar suggestions.

If you do make some changes, would you also add the word can to the sentence 

> If you use BridgeStan in your research, please consider citing [our JOSS paper](https://joss.theoj.org/papers/10.21105/joss.05236) and letting us know so we [can] list your project here.

in the Section Research using BridgeStan of bridgestan/README.md?

File: docs/languages.rst (hunk 2 above)
Comment (WardBrian) [reply docs/languages.rst:14]: Rewrote the whole sentence to "If BridgeStan's bindings to your favorite language are missing one of these features, we welcome contributions!"

File: docs/languages/rust.rst (hunk 3 above)
Comment (WardBrian) [reply docs/languages/rust.rst:17]: Unlike Julia, etc, in Rust we do make the user actually call a separate function to download, so I think "can be" is more accurate

**Comment by bob-carpenter** [comment]
Just stopping by to say I approve of the level of grammatical pedantry on both sides!  If you want one more, I don't like "Note that" because it's redundant.

And yes please on the citation nudge.

**Pull request merged**